### PR TITLE
Update homework.py

### PR DESCRIPTION
**Pull request by BulgakovMax**
First pull request

### CODE REVIEW

File: homework.py [hunk 1]
```diff
@@ -56,11 +53,11 @@ def multiple_ints(first_value: int, second_value: int) -> int:
         Product of elements
     """
     pass
-    try:
-        if (isinstance(first_value, int) and isinstance(second_value, int)) == True:
-            return first_value + second_value
-    except ValueError:
-        print("invalid literal should  enter integer")
+    if (isinstance(first_value, int) and isinstance(second_value, int)) == True:
```
Comment (alchemixXx) [inline homework.py]: unnecessary check "== True" : 
'if' -block will work only if both values equal True.

File: homework.py [hunk 2]
```diff
@@ -156,19 +148,19 @@ def alphabet() -> dict:
...
     """
     pass
     n = len(data)
     for i in range(n):
         for j in range(0, n - i - 1):
             if data[j] > data[j + 1]:
                 data[j], data[j + 1] = data[j + 1], data[j]
-    return data
\ No newline at end of file
+    return data
```
Comment (alchemixXx) [inline homework.py:158]: add empty line in the end of file

File: homework.py (hunk 1 above)
Comment (BulgakovMax) [reply homework.py]: Fixed

File: homework.py (hunk 2 above)
Comment (BulgakovMax) [reply homework.py:158]: Fixed

File: homework.py [hunk 3]
```diff
@@ -124,7 +111,8 @@ def some_loop_exercise() -> list:
     Use loop to create list that contain int values from 0 to 12 except 6 and 7
     """
     pass
-    return [i for i in range(13) if i not in [6, 7]]
+    return [i for i in range(13) if i not in [6, 7]]\
```
Comment (alchemixXx) [inline homework.py]: What is it in the end of line?)

**Review by alchemixXx — APPROVED** [review]

File: homework.py (hunk 3 above)
Comment (BulgakovMax) [reply homework.py]: Fixed

**Pull request merged**